miden-tx/src/testing/tx_context.rs

```diff
@@ -77,14 +143,6 @@ impl TransactionContext {
         InputNotes::new(self.mock_chain.available_notes().clone()).unwrap()
     }
 
-    pub fn tx_args(&self) -> &TransactionArgs {
-        &self.tx_args
-    }
-
-    pub fn set_tx_args(&mut self, tx_args: TransactionArgs) {
-        self.tx_args = tx_args;
-    }
```

Comment:
The args are no longer created when instantiating the context, now it is created just before executing the transaction, this too is because of the tx script.